src/main/java/nextstep/subway/applicaion/LineService.java

```diff
@@ -19,43 +16,39 @@
 @Service
 public class LineService {
     private final LineRepository lineRepository;
-    private final StationService stationService;
 
-    public LineResponse saveLine(LineRequest request) {
-        validateDuplicateLineName(request.getName());
+    public LineResponse saveLine(LineRequest lineRequest, StationsDto stationsDto) {
+        validateDuplicateLineName(lineRequest.getName());
 
-        Line line = lineRepository.save(new Line(request.getName(), request.getColor()));
-        Section section = createSection(request, line);
+        Line line = new Line(lineRequest.getName(), lineRequest.getColor());
+        Section section = createSection(line, stationsDto, lineRequest.getDistance());
         line.getSections().add(section);
```

Comment:
섹션을 LineService에서 생성하여 line에 넘겨줄 경우 LineService는 Section에 의존하게 됩니다.
제거할 수 있는 의존을 최대한 제거하기 위해서는 Section을 생성하는 로직을 line 내부로 옮기는 것을 추천드립니다.
즉, LineService가 Section의 존재를 모르게 하면 LineService가 Section에 의존하지 않게 됩니다.
아래와 같은 형태가 될 수 있도록 리팩터링 해보세요 :) 

```suggestion
        line.addSection(upStation, downStation, lineRequest.getDistance());
```